<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_appoint">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>334</y>
      <width>581</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Appointments</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_1">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit_1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>118</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>118</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>118</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>118</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>118</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Use Keyboard</string>
     </property>
    </widget>
   </widget>
   <widget class="QCalendarWidget" name="cw">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>340</y>
      <width>371</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>981</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_main">
       <property name="title">
        <string>Main task</string>
       </property>
       <widget class="QCheckBox" name="checkBox_1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_1">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>140</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>180</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_6">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>220</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_sub">
       <property name="title">
        <string>Sub task</string>
       </property>
       <widget class="QCheckBox" name="checkBox_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_7">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_8">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_9">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_10">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>140</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_11">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>180</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_12">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>220</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_memo">
       <property name="title">
        <string>Memo</string>
       </property>
       <widget class="QPlainTextEdit" name="plainTextEdit_memo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>281</width>
          <height>231</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_save">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>260</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>SAVE</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_open">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>260</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>OPEN</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
